schemas/com.snowplowanalytics.snowplow/ua_parser_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "http://json-schema.org/schema#",
```

Comment:
Schema should be "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",